week-1/interpret/to-pounds.js

```diff
@@ -1,21 +1,21 @@
-const penceString = "399p";
+const penceString = "399p"; //initializes a string variable with the value "399p"
 
 const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
-);
+);    // Initializes a new variable to remove the last character from the string
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); // Initializes a new variable to pad the whole number
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
-);
+); // New variable to pad the number to make value in pounds 
```

Comment:
This expression doesn't do any padding on the string. Can you think what is happening with the function `substring(..)` and the numbers given as arguments?

Reply:
I have modified my answer